src/sentry/api/endpoints/sentry_apps_stats.py

```diff
@@ -12,18 +12,19 @@ class SentryAppsStatsEndpoint(SentryAppsBaseEndpoint):
     permission_classes = (SuperuserPermission,)
 
     def get(self, request):
-        sentry_apps = SentryApp.objects.filter(installations__date_deleted=None).annotate(
-            Count("installations")
-        )
-        results = []
-        for app in sentry_apps:
-            results.append(
-                {
-                    "id": app.id,
-                    "slug": app.slug,
-                    "name": app.name,
-                    "installs": app.installations__count,
-                }
-            )
-
-        return Response(results)
+        sentry_apps = SentryApp.objects \
+            .filter(installations__date_deleted=None) \
+            .annotate(Count("installations")) \
+            .order_by()
+
+        apps = [{
+            "id": app.id,
+            "slug": app.slug,
+            "name": app.name,
+            "installs": app.installations__count,
+        } for app in sentry_apps]
```

Comment:
Don't you still load all the data into memory? Should you do the slice first?

Reply:
Yea, good call. I moved the up before it "serializes" things, which should use a `LIMIT` instead.